draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -9,11 +9,12 @@ type Props = {
   position?: Position
   text: string
   children?: React.ReactNode
+  className?: string
 }
 
 const Tooltip = (props: Props) => (
   <span
-    className={classnames(styles.tooltipWrap, {
+    className={classnames(styles.tooltipWrap, props.className, {
```

Comment:
I know this one breaks the rules, but I'm not really sure how else to fix the issue I'm having with the tooltip.

Basically I want the width and height to be 100%, as so I can appropriately align the content inside. See the `.headerRowCellTooltip` class definition above to see what I mean.

Reply:
Alternatively, I could pass down 4 new properties to tooltip: flex, width, display, align-items